### PR TITLE
fix: add missing tests

### PR DESCRIPTION
**Pull request by aziiee**
This will also make the required changes after bumping `faker`.

### CODE REVIEW

File: tests/acceptance/data-table-test.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ module("Acceptance | data-table", function (hooks) {
     );
   });
 
-  test("pagination", async function (assert) {
+  skip("pagination", async function (assert) {
```
[the remark this answers is not in the document]
Comment (aziiee) [reply tests/acceptance/data-table-test.js:65]: @velrest yes, everything looks just fine. I can't find any issues when running the test locally

[the remark this answers is not in the document]
Comment (aziiee) [reply tests/acceptance/data-table-test.js]: @velrest the issue has been fixed in #296

**Review by czosel — APPROVED** [review]
LGTM

**Pull request merged**